crypto/source_address_token.go

```diff
@@ -58,7 +58,11 @@ const stkKeySize = 16
 const stkNonceSize = 16
 
 // NewStkSource creates a source for source address tokens
-func NewStkSource(secret []byte) (StkSource, error) {
+func NewStkSource() (StkSource, error) {
```

Comment:
True, but I still think it makes sense to keep (and probably even refresh) them together. Both are secrets that will have to live across server restarts, and be shared by multiple servers in a load-balancing setup. FWIW, gQUIC also keeps them together.